### ISSUE TITLE
Deployment fails when supplying an empty array for subscriptions

### ISSUE THREAD

**Issue by tw3lveparsecs** [tw3lveparsecs/azure-bicep-management-groups#1]
example below

``` bicep
targetScope = 'tenant'

param deploymentName string = concat('managementGroups', utcNow())

module managementGroupGlobal '../main.bicep' = {
  name: concat(deploymentName, 'managementGroupGlobal')
  params: {
    managementGroupDisplayName: 'MyOrganisation'
    managementGroupId: 'gbl-org-mgp'
    subscriptionIds: []
  }
}
```

**Comment by tw3lveparsecs** [comment]
this is a current limitation of Bicep and will be resolved with bicep v0.4 https://github.com/Azure/bicep/issues/1754

**Comment by tw3lveparsecs** [comment]
solved with Azure/bicep#1754
